Vagrantfile

```diff
@@ -76,7 +76,7 @@ end
 
 Vagrant.configure('2') do |config|
   config.vm.box = 'bento/centos-6.7'
-  config.ohai.primary_nic = 'eth1'
+  config.ohai2.primary_nic = 'eth1'
```

Comment:
Log an issue so that we handle multiple NICs properly. It'd be nice to not have to rely on this...

It would also be nice to manage address registration for indexers as well...

But that's scope creep.

Reply:
Writing a new library to handle the work that used to be done by the ohai vagrant plugin.